lib/views/header-view.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import path from 'path';
+
+export default class HeaderView extends React.Component {
+  static propTypes = {
+    currentProject: PropTypes.string,
+    projectPaths: PropTypes.arrayOf(PropTypes.string),
+
+    handleProjectSelect: PropTypes.func.isRequired,
+  }
+
+  render() {
+    return (
+      <header className="github-Project">
+        <select className="github-Project-path input-select"
+          value={this.props.currentProject ? this.props.currentProject : undefined}
+          onChange={this.props.handleProjectSelect}>
+          {this.renderProjects()}
+        </select>
+      </header>
+    );
+  }
+
+  renderProjects = () => {
```

Comment:
This doesn't need to be a property-set method because it isn't called in a way that explicitly needs `this` bound:

```suggestion
  renderProjects() {
```